.env.example

```diff
@@ -8,7 +8,7 @@ MEDIA_URL=/media/
 MEDIA_ROOT=data/media/
 INTERNAL_IPS=["127.0.0.1","localhost"]
 SITE_URL=http://localhost:8000
-EMAIL_URL=dummymail://
+EMAIL_URL=smtp://
```

Comment:
does this throw errors when used as is? maybe we should keep dummymail in the example to allow for local development without changes

Reply:
Would be fine with that. As is no errors are thrown because emails are only sent if Debug=FALSE

Reply:
ah, I see. Would still vote for the dummymail default as we will integrate other features that send mails (minutes reminder etc.) which would also send with DEBUG=True